test/driver-test/driver-test.cpp

```diff
@@ -0,0 +1,306 @@
+/*
+* driver-test.cpp
+* Nick DeMarco
```

Comment:
I believe including my name (second) is appropriate because I have made significant contribution to the design of and implementation of this file's contents.